MVVM Module/Source/Feed UI/Controllers/FeedViewController.swift

```diff
@@ -27,11 +29,15 @@ public final class FeedViewController: UITableViewController, UITableViewDataSou
 		title = viewModel?.title
 		viewModel?.onLoadingStateChange = { [weak self] isLoading in
 			if isLoading {
+				self?.errorView.hideMessage()
 				self?.refreshControl?.beginRefreshing()
 			} else {
 				self?.refreshControl?.endRefreshing()
 			}
 		}
+		viewModel?.onFeedLoadError = { [weak self] _ in
+			self?.errorView.show(message: Localized.Feed.loadError)
+		}
```

Comment:
The ViewModel should decide which message to show. See my other comment.
```suggestion
		viewModel?.onFeedLoadError = { [weak self] message in
			if let message = message {
				self?.errorView.show(message: message)	
			} else {
				self?.errorView.hideMessage()
			}
		}
```

Reply:
Ok!